core/services/job/models_test.go

```diff
@@ -322,6 +324,9 @@ func TestWorkflowSpec_Validate(t *testing.T) {
 			w := &WorkflowSpec{
 				Workflow: tt.fields.Workflow,
 			}
+			sum := sha256.New()
+			sum.Write([]byte("test"))
+			w.WorkflowID = fmt.Sprintf("%x", sum.Sum(nil))
```

Comment:
I think you can remove this altogether; the only reason we set this is to pass the assertion on line 333 (because Validate sets this)

Both can go as it shouldn't be relevant

Reply:
Validate fails if the size of the workflow id isn't right.  Wasn't sure if it was ok to remove that.